src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -144,7 +145,9 @@ export function createStaticCommandBarButtons(
         commandButtonLabel: label,
         ariaLabel: label,
         hasPopup: true,
-        disabled: selectedNodeState.isDatabaseNodeOrNoneSelected(),
+        disabled: useSelectedNode.getState().isQueryCopilotCollectionSelected()
```

Comment:
Can be simplified to `useSelectedNode.getState().isQueryCopilotCollectionSelected() || selectedNodeState.isDatabaseNodeOrNoneSelected()`